frontend/src/components/BarChartWidget.tsx

```diff
@@ -126,7 +126,8 @@ const BarChartWidget = (props: Props) => {
               type="number"
               domain={[
                 (dataMin: number) => 0,
-                (dataMax: number) => dataMax + Math.floor(dataMax * 0.2),
+                (dataMax: number) =>
```

Comment:
I think this is not exactly what we want here. Please refer Jason for clarification.